libs/util/src/index.ts

```diff
@@ -0,0 +1,6 @@
+import type { RestManager } from '@cordis/rest';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export function makeRestUtils(restRouter: RestManager) {
```

Comment:
```suggestion
export const makeRestUtils = (rest: RestManager) => {
```

Prefer arrow functions.